### DISCUSSION TITLE
Level of background w.r.t. mass

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#36] [Comments_HIG_23_001]
> - What is the typical level of background for a signal, for masses of 500, 750, and 1000 GeV? (e.g. background under 68% of the signal)

Get these values.

**Comment by jialin-guo1** [comment]
@ram1123 
**computation** : n_bkg/(n_bkg+n_signal) in signal region. signal = ggh+vbf

**mass 500**:
resolved: 42.02%
merged: 9.77%

**mass 750**:
resolved: 83.99%
merged:  14.41%

**mass 1000**:
resolved: 96.36%
merged: 28.18%

**Reply by ram1123** [reply]
These numbers seem a bit suspicious as the background contamination should decrease instead of increasing.  I will try to check carefully tomorrow.

**Reply by jialin-guo1** [reply]
maybe because of cross section decrease in signal sample.
cross section of mass 500: 3.131
cross section of mass 750: 0.445
cross section of mass 1000: 0.1023

**Reply by jialin-guo1** [reply]
@ram1123 
updata to set all cross section of signal to 1
**computation** : n_bkg/(n_bkg+n_signal) in signal region. signal = ggh+vbf 

**mass 500**:
resolved: 56.78%
merged: 29.52%

**mass 750**:
resolved: 64.60%
merged:  6.42%

**mass 1000**:
resolved: 74.77%
merged: 8.75%

could you confirm that I need to count all range of bkg into n_bkg when I compute contamination for different mass point. I remember that you said, for example, that if we compute limit on m1000 then the bkg around m1000 will contribute more than bkg far away from m1000

**Reply by ram1123** [reply]
yes, but here we need the inclusive numbers.

**Reply by ram1123** [reply]
Can you also get these numbers for 1500 and 2000 GeV?
